scss/modules/_nav.scss

```diff
@@ -75,6 +85,7 @@
     @include transform(translateX(0));
     background: none;
     border: none;
+    min-height: 0;
```

Comment:
Properties should be ordered background, border, display, float, min-height, padding, position, top